docs/src/content/docs/MCP.md

```diff
@@ -32,29 +32,75 @@ Make sure to also properly configure the `tsconfig.json` and `tsconfig.app.json`
 
 <https://ui.shadcn.com/docs/installation/vite>
 
+Initialize ShadCN:
+
+```sh
+npx shadcn@latest init
+```
+
 ## Setup the registry MCP
 
+Follow the instructions to create a new MCP server named `shadcn` that uses the `shadcn@latest mcp` command to fetch components.
+
+### Claude Code
+
+Create a file named `.mcp.json` and copy the following content into it:
+
+```json
+{
+  "mcpServers": {
+    "shadcn": {
+      "command": "npx",
+      "args": ["shadcn@latest", "mcp"]
+    }
+  }
+}
+```
+
+### Cursor
+
 Create a file named `.cursor/mcp.json` and copy the following content into it:
 
 ```json
 {
     "mcpServers": {
-        "shadcn-admin-kit": {
+        "shadcn": {
+            "command": "npx",
+            "args": [
+                "shadcn@latest",    
+                "mcp"
+            ]
+        }
+    }
+}
+```
+
+### VS Code
+
+Create a file named `.vscode/mcp.json` and copy the following content into it:
+
+```json
+{
+    "servers": {
+        "shadcn": {
             "command": "npx",
             "args": [
-                "-y",
-                "shadcn@canary",
-                "registry:mcp"
-            ],
-            "env": {
-                "REGISTRY_URL": "https://marmelab.com/shadcn-admin-kit/r/registry.json"
-            }
+                "shadcn@latest",    
+                "mcp"
+            ]
         }
     }
 }
 ```
 
-This will create a new MCP server named `shadcn-admin-kit` that uses the `shadcn@canary registry:mcp` command to fetch components from the Shadcn Admin Kit registry.
+### Update shadcn
+
+Then, update your `components.json` file by adding the shadcn-admin-kit registry:   
+```
+"registries": {
+    "@shadcn-admin-kit": "https://marmelab.com/shadcn-admin-kit/r/{name}.json"
+}
+```
 
 ## Start prompting
```

Comment:
Now that a project can have multiple registries, I suggest we edit the example prompts with the following:

```md
Some examples of prompts you can use:

- "show me all available components in the shadcn-admin-kit registry"
- "init this project using the shadcn-admin-kit registry"
- "create a new admin using the shadcn-admin-kit registry and declare 3 resources in it: posts, comments and users"
- "customize each resource to add a matching icon using the lucide library"
```